sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -196,8 +196,11 @@ enum PropKind : uint32_t {
   Balanced = 55,
   InvocationCapacity = 56,
   ResponseCapacity = 57,
+  RangeKernel = 58,
```

Comment:
Do you still need this?  Isn't this related to the (now removed) `range` property?

Reply:
Yes, I'll remove that in the next update.